docs/getting-started/03-deploy-microservice.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Deploy a microservice
+type: Getting Started
+---
+
+You will now start by deploying a standalone [`orders-service`](https://github.com/kyma-project/examples/blob/master/http-db-service/README.md) microservice in the `orders-service` Namespace. This microservice will act as a link between the external application and the Redis service and we will build the whole end-to-end flow around it.
+
+In this guide you will create:
+
+- Deployment in which you specify the configuration of your microservice.
+- Kubernetes Service through which your microservice will communicate with other resources on the cluster.
+
+## Steps
+
+### Create the Deployment
+
+Create a [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) that provides the microservice definition and enables you to run it on the cluster. The Deployment uses the `eu.gcr.io/kyma-project/pr/orders-service:PR-162` image. This Docker image exposes the `8080` port on which the related Service is listening.
+
+Follow these steps:
+
+<div tabs name="steps" group="deploy-microservice">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Apply the microservice definition to the `orders-service` Namespace on your cluster:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/orders-service/deployment/orders-service-deployment.yaml
+```
+
+2. Check if the Deployment was created. The correct Deployment status should set **readyReplicas** to `1`:
+
+```bash
+kubectl get deployment orders-service -n orders-service -o=jsonpath="{.status.readyReplicas}"
+```
+
+</details>
+<details>
+<summary label="ui">
+UI
+</summary>
+
+1. Create `orders-service-deployment.yaml` on your machine containing [this Deployment definition](https://raw.githubusercontent.com/kyma-project/examples/master/orders-service/deployment/orders-service-deployment.yaml).
```

Comment:
```suggestion
1. On your machine, create `orders-service-deployment.yaml` containing [this Deployment definition](https://raw.githubusercontent.com/kyma-project/examples/master/orders-service/deployment/orders-service-deployment.yaml).
```
**Reason:**
I restructured that to avoid ambiguity (i.e. the deployment definition is contained by the `.yaml` file or the machine?). 
I thought about wording it like this first: `1. Create `orders-service-deployment.yaml` containing [this Deployment definition](https://raw.githubusercontent.com/kyma-project/examples/master/orders-service/deployment/orders-service-deployment.yaml) on your machine.` but this is still a bit ambiguous.

Reply:
Ok. I change it this way (both for Deployment and Service):
```
1. On your machine, create  ...
2. Back in the Console UI, go to ... 
```